### ISSUE TITLE
Issue with contents of a storage tank

### ISSUE THREAD

**Issue by EwenH** [openstreetmap/iD#4169]
Hi,
   Hopefully a quick fix with an issue with selecting the contents of a storage tank. Whilst the correct tag "contents" is used in the storage tank default tags, it becomes a singular "content" when adding the tag.

1. Open ID editor 
2. Create a basic area
3. From the left hand side search, select "Storage Tank"
4. Select "water" from the contents 
5. water is added to the list of tags as a value as expected however "contents" has become "content" as the tag << Failure

![image](https://user-images.githubusercontent.com/4280493/28426309-1f84c3cc-6db6-11e7-893f-e3b9fc5c7560.png)

**Comment by althio** [comment]
`content` is the prefered tag according to [wiki.osm.org:Key:content](http://wiki.openstreetmap.org/wiki/Key:content)
So `content=water` is correct.

In iD "Contents" is here not a hard-rule tag in the data.
It is only used as a label in plain english language in the interface.
I guess both "Contents" and "Content" are correct.

**Comment by EwenH** [comment]
Thanks Althio however the validator keepright.at states that "content" should be "contents" and it would be useful however I now see that OSM wiki stats that content is correct. Perhaps just change the label

**Comment by bhousel** [comment]
Sure, we can change the caption from "Contents" to "Content" if that would be clearer.
 
The field captions do not always exactly match the preset key, but in this case they are both acceptable in English, so we might as well call it what the key is called to avoid confusion.
